src/riak_pb_ts_codec.erl

```diff
@@ -36,7 +36,6 @@
          encode_rows/1,
          encode_cells/1,
          decode_rows/1,
-         decode_row/1,
```

Comment:
Really? I recompiled my clone of Riak on the `end-to-end/timeseries` branch while using this branch of `riak_pb` and it worked fine. Grepping `riak-erlang-client` on the `end-to-end/timeseries` branch doesn't turn anything up either:

```
lbakken@brahms ~/Projects/basho/riak-erlang-client (end-to-end/timeseries=)
$ git pull && git grep '\<decode_row\>'
Current branch end-to-end/timeseries is up to date.
```